concepts.wip/integer-introduction/about.md

```diff
@@ -0,0 +1,78 @@
+# Introduction
+
+Integer literals in Julia are represented in the usual way.
```

Comment:
What's the unusual way?

Reply:
Well, the target audience is programmers, so I'm assuming they've done it before in some language. Is there a more descriptive way to describe how to write an integer?